src/EnergyPlus/SetPointManager.cc

```diff
@@ -250,9 +278,6 @@ void GetSetPointManagerInputData(EnergyPlusData &state, bool &ErrorsFound)
     using ScheduleManager::GetScheduleIndex;
 
     // Locals
-    // SUBROUTINE PARAMETER DEFINITIONS:
-    const char *RoutineName("GetSetPointManagerInputs: "); // include trailing blank space
-
```

Comment:
we deleted this variable so that we don't use std::string{}, as recommended by Amir. In the following changes, all the std::string{} are replaced with the format fuction.